.github/workflows/standalone-fleet-ci.yml

```diff
@@ -0,0 +1,119 @@
+name: E2E Standalone Fleet
+
+on:
+  schedule:
+    - cron:  '0 5 30 * *'
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
```

Comment:
nit:

can we change the name of this or is this a fixed name? if possible to change, we should make this `debug` or `enable_debug` instead of `debug_enabled`

Reply:
Hm yes, I copied that from the tmate action docs. `enable_debug` is much nicer, but it doesn't really enable debug.. it enables tmate for debugging. It's not putting any fleet component into debug mode, or run the workflow with debug. Maybe just `enable_tmate`?